docs/streams/upgrade-guide.html

```diff
@@ -126,7 +127,11 @@ <h3><a id="streams_api_changes_120" href="#streams_api_changes_120">Streams API
       The new class <code>To</code> allows you to send records to all or specific downstream processors by name and to set the timestamp for the output record.
       Forwarding based on child index is not supported in the new API any longer.
     </p>
-
+    <p>
+        <a href="https://cwiki.apache.org/confluence/x/DVyHB">KIP-284</a> changed the retention time for repartition topic by setting its default value to Long.MAX_VALUE.
+        <li>Use purge data API to keep storage small</li>
```

Comment:
No need to list markup. Would just add a second sentence: `Instead of relying on data retention Kafka Streams uses the new purge data API to delete consumed data from those topics and to keep used storage small now.`